src/tensors/truncation.jl

```diff
@@ -72,7 +72,9 @@ function _truncate!(v::AbstractVector, trunc::TruncationCutoff, p::Real = 2)
     if dtrunc === nothing
         dtrunc = 0
     end
-    dtrunc < length(v) && trunc.addone && (dtrunc+=1)
+    for i in 1:trunc.add_back
+	dtrunc < length(v) && (dtrunc+=1)
+    end
```

Comment:
`dtrunc = min(dtrunc + trunc.add_back, length(v))` as one-liner?

Reply:
Yes this would be cleaner !